Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -18,14 +18,6 @@ open class ChatChannelListVC: _ViewController,
     public var controller: ChatChannelListController!
 
     private var isPaginatingChannels: Bool = false
-
-    open private(set) lazy var loadingIndicator: UIActivityIndicatorView = {
```

Comment:
This would be a breaking change, right?

Reply:
Oh yes... good point 🤔 We need to think about this a bit better

Reply:
🤔. Should I leave it and add a message that would be deprecated soon? In that case I'd probably have to provide a boolean to allow them implementers to choose between one or the other. Is that ok? or any other ideas?

Reply:
Yes, I think opt in for the new logic should be provided (enabled by default in our demo app). Deprecating the `loadingIndicator` is also a good idea.

Reply:
Yes, I think that makes sense indeed. These new loading states should be opt-in. Or we might break other customers. Especially since there's a high chance they have their own loading states already. 👍

Reply:
Cool, we can go with this approach. Just please change the default behaviour in the SDK to use the loading indicator (see the other comment) and enable this from the demo app.

Reply:
Done 👍🏻